pom.xml

```diff
@@ -158,4 +158,20 @@
 			<url>https://repo.jenkins-ci.org/public/</url>
 		</pluginRepository>
 	</pluginRepositories>
+	<build>
```

Comment:
No you don't need it.
You will need to update the Jenkinsfile first to be able to use a newer Jenkins version.